randovania/games/prime3/logic_database/Bryyo - Fire.txt

```diff
@@ -448,9 +437,22 @@ Extra - asset_id: 13296637487518400273
                   Combat (Beginner) and Damage ≥ 100
                   # https://youtu.be/4OggF458OMI&t=65
                   After Hangar Bay Ship Upgrade and Before Temple of Bryyo Turret and Combat (Intermediate) and Use Ship Missile (Command Visor and Ship Missile)
-          Combat (Intermediate) and Use Screw Attack (Space Jump)
+          Combat (Advanced) and Use Screw Attack (Space Jump)
+          # Collect Gel Pickup to Activate the Elevator to Kill Rundas
+          After Temple of Bryyo Gel Pickup Collected and Knowledge (Expert)
```

Comment:
I wonder if Expert is too high here. Could Advanced work?

Reply:
My thought process for it being Expert was more due to how obscure this is rather than how difficult.
If there are more edge cases similar to this to compare to, then I'd probably have a better idea to go off of. 
Otherwise, I think Expert should be fine.